lib/diagnostic.rb

```diff
@@ -1,44 +1,53 @@
 # frozen_string_literal: true
-
+list = 1..10
 # Ruby enumerable diagnostic
 class Diagnostic
   # Question 1
   # Write ruby code to return the longest entry in the list supplied.
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    list.max
   end
-
+  longest(list)
   # Question 2
   # Write ruby code to return the sum of entries in the
   # list supplied (i.e. apply the `+` operator).
   # Assume each entry implements a `+` method.
   def sum(list)
-    # Your code goes here
+    list.reduce(:+)
   end
+  sum(list)
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the list of numbers are odd
-  def some_odd(numbers)
-    # Your code goes here
+  def some_odd(list)
+    list.any? { :odd? }
```

Comment:
Looks like you gave it a block, should have been parens:

`numbers.any?(&:odd?)`